src/data/locations/death_mountain_crater.json

```diff
@@ -1,726 +1,1215 @@
+      "foundItem": "",
+      "merchant": false,
+      "price": 0,
+      "walletTier": 0,
+      "shuffled": true,
+      "shuffleRules": []
+    },
+    "DMC Wall Freestanding PoH": {
+      "area": "Death Mountain Crater",
+      "lKey": "",
+      "alias": "Item In Wall",
+      "type": "item",
+      "settings": [],
+      "visible": true,
+      "check": "",
+      "foundItem": "",
+      "merchant": false,
+      "price": 0,
+      "walletTier": 0,
+      "shuffled": true,
+      "shuffleRules": []
+    },
+    "DMC Deku Scrub": {
+      "area": "Death Mountain Crater",
+      "lKey": "",
+      "alias": "Child Scrub",
+      "type": "item",
+      "settings": [
+        {
+          "setting": "Scrubsanity",
+          "value": "Affordable",
+          "required": false
+        },
+        {
+          "setting": "Scrubsanity",
+          "value": "Random",
+          "required": false
+        },
+        {
+          "setting": "Scrubsanity",
+          "value": "Vanilla",
+          "required": false
+        }
+      ],
+      "visible": true,
+      "check": "",
+      "foundItem": "",
+      "merchant": false,
+      "price": 0,
+      "walletTier": 0,
+      "shuffled": true,
+      "shuffleRules": []
+    },
+    "Bolero of Fire": {
+      "area": "Death Mountain Crater",
+      "lKey": "",
+      "alias": "Bolero of Fire",
+      "type": "song",
+      "settings": [],
+      "visible": true,
+      "check": "",
+      "foundItem": "",
+      "merchant": false,
+      "price": 0,
+      "walletTier": 0,
+      "shuffled": true,
+      "shuffleRules": []
+    },
+    "DMC Volcano Freestanding PoH": {
+      "area": "Death Mountain Crater",
+      "lKey": "",
+      "alias": "Volcano Item",
+      "type": "item",
+      "settings": [],
+      "visible": true,
+      "check": "",
+      "foundItem": "",
+      "merchant": false,
+      "price": 0,
+      "walletTier": 0,
+      "shuffled": true,
+      "shuffleRules": []
+    },
+    "DMC GS Bean Patch": {
+      "area": "Death Mountain Crater",
+      "lKey": "",
+      "alias": "Bean Patch GS",
+      "type": "skulltula",
+      "settings": [
+        {
+          "setting": "Show Unshuffled Skulls",
+          "value": "Yes",
+          "required": false
+        },
+        {
+          "setting": "Tokensanity",
+          "value": "Overworld",
+          "required": false
+        },
+        {
+          "setting": "Tokensanity",
+          "value": "All",
+          "required": false
+        }
+      ],
+      "visible": true,
+      "check": "",
+      "foundItem": "",
+      "merchant": false,
+      "price": 0,
+      "walletTier": 0,
+      "shuffled": true,
+      "shuffleRules": [
+        {
+          "setting": "Tokensanity",
+          "value": "Overworld",
+          "required": false
+        },
+        {
+          "setting": "Tokensanity",
+          "value": "All",
+          "required": false
+        }
+      ]
+    },
+    "DMC Plant Bean Patch": {
```

Comment:
Can you remove the events from the location lists? It messes with the counter.

Also applies to the Plant Bean and Scarecrow events in other files.

I'd also be open to making these an option as long as they aren't included in the total location count.

Reply:
is there a way to exclude them from the total count in addition to moving them under a new setting? haven't looked too deeply into the gui source code itself. i can separate them into a setting called 'events' or something similar. it would affect plant beans, scarecrow, and also the ones i added for the epona race and waking up talon in the master carpenter's house.

that last one I added because if there are no checks inside that interior it immediately gets hidden and can be hard to find where it is again on interior shuffle. it's also the only thing inside that interior that has any relevance to any play through.

if i made this 'events' setting i would probably also want to add the selling of masks to the guard, skull kid, etc as separate things. stuff under this umbrella becomes less about tracking item locations and more like reminders that this is something you might need to do while in this area.